service/chartconfig/v1/helm/helm.go

```diff
@@ -67,11 +67,41 @@ func (c *Client) GetReleaseContent(customObject v1alpha1.ChartConfig) (*Release,
 		return nil, microerror.Mask(err)
 	}
 
-	release := &Release{
+	content := &ReleaseContent{
 		Name:   resp.Release.Name,
 		Status: resp.Release.Info.Status.Code.String(),
 		Values: values.AsMap(),
 	}
 
-	return release, nil
+	return content, nil
+}
+
+// GetReleaseHistory gets the current installed version of the Helm Release.
+func (c *Client) GetReleaseHistory(customObject v1alpha1.ChartConfig) (*ReleaseHistory, error) {
+	var releaseVersion string
+
+	releaseName := key.ReleaseName(customObject)
+
+	resp, err := c.helmClient.ReleaseHistory(releaseName, helmclient.WithMaxHistory(1))
+	if err != nil && IsReleaseNotFound(err) {
```

Comment:
Would be cool to keep the current style. 

```
if IsReleaseNotFound(err) {
  // 
} else if err != nil {
  // 
}
```

Also `IsReleaseNotFound` should not return true if `err` is `nil`.

Reply:
Good point. Done.